requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
-pytest
+pytest>=2.8
```

Comment:
I see the version conflict exception in the Travis build, so I guess it's the minimum version supported by `pytest-flake8`. I wonder why that doesn't specify the version itself.